libraries/chain/combined_database.cpp

```diff
@@ -112,7 +112,7 @@ namespace eosio { namespace chain {
             if (!status.ok())
                throw std::runtime_error(std::string{ "database::database: rocksdb::DB::Open: " } + status.ToString());
             auto rdb        = std::shared_ptr<rocksdb::DB>{ p };
-            return std::make_unique<rocks_db_type>(eosio::session::make_session(std::move(rdb)));
+            return std::make_unique<rocks_db_type>(eosio::session::make_session(std::move(rdb), 1024));
```

Comment:
Can you add a comment or use a constexpr about 1024?

I see below it is max_iterators. Then there is a system consensus limits (whose type kv_database_config defines  max_key_size, max_value_size, max_iterators). I think you should use max_iterators from limits.